egs/librispeech/ASR/pruned_transducer_stateless7/train.py

```diff
@@ -368,6 +368,13 @@ def get_parser():
         help="Whether to use half precision training.",
     )
 
+    parser.add_argument(
```

Comment:
I believe there should be no cost to printing this information, and it should be printed very rarely, so I don't think it is necessary to have a command line arg, just make it true.